src/BlazorUI/Bit.BlazorUI/Components/Progress/Progress/BitProgress.scss

```
@@ -6,8 +6,26 @@
     font-size: $tg-fs-sm;
     font-family: $tg-font-family;
     --bit-prb-bar-color: #{$clr-pri};
+    --bit-prb-bar-on-color: #{$clr-pri-text};
+    // Where the circular stroke starts, and which way round it goes. A <circle> path starts at three
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)